test/e2e/node/runtimeclass.go

```diff
@@ -115,6 +115,44 @@ var _ = ginkgo.Describe("[sig-node] RuntimeClass", func() {
 		framework.ExpectEqual(nodeSelector, pod.Spec.NodeSelector)
 		gomega.Expect(pod.Spec.Tolerations).To(gomega.ContainElement(tolerations[0]))
 	})
+
+	ginkgo.It("should run a Pod requesting a RuntimeClass with scheduling without taints [NodeFeature:RuntimeHandler] ", func() {
+		nodeName := scheduling.GetNodeThatCanRunPod(f)
+		nodeSelector := map[string]string{
+			"foo":  "bar",
+			"fizz": "buzz",
+		}
+		scheduling := &nodev1beta1.Scheduling{
+			NodeSelector: nodeSelector,
+		}
+
+		ginkgo.By("Trying to apply a label on the found node.")
+		for key, value := range nodeSelector {
+			framework.AddOrUpdateLabelOnNode(f.ClientSet, nodeName, key, value)
+			framework.ExpectNodeHasLabel(f.ClientSet, nodeName, key, value)
+			defer framework.RemoveLabelOffNode(f.ClientSet, nodeName, key)
+		}
+
+		ginkgo.By("Trying to create runtimeclass and pod")
+		runtimeClass := newRuntimeClass(f.Namespace.Name, "non-conflict-runtimeclass", framework.TestContext.ContainerRuntime)
+		runtimeClass.Scheduling = scheduling
+		rc, err := f.ClientSet.NodeV1beta1().RuntimeClasses().Create(context.TODO(), runtimeClass, metav1.CreateOptions{})
+		framework.ExpectNoError(err, "failed to create RuntimeClass resource")
+
+		pod := e2enode.NewRuntimeClassPod(rc.GetName())
+		pod.Spec.NodeSelector = map[string]string{
+			"foo": "bar",
```

Comment:
foo bar is pretty common, so it's possible there could be a conflict with another test. Probably best to generate some random label. Maybe use the test namespace as the label?

Reply:
fixed. is it what did you have in mind?

Reply:
decided to go with UUID - so tests wouldn't interfere